portfolioStyles.css

```diff
@@ -197,12 +205,46 @@ nav img{
 	margin-bottom: 5%;
 }
 
+.email{
+	padding: 20px;
+	/*margin: 15px;*/
```

Comment:
I don't think Charlie is a fan of commented out code in the final edit? 
I have loads of comments in mine lol... we'll see how that goes.